### DISCUSSION TITLE
Stable Release

### DISCUSSION THREAD

**Discussion by vanhoutenbos** [SixLabors/ImageSharp.Drawing#153] [General]
Hi!

We have been using this library for quite a while now and I am wondering if there are any plans for a stable release.
We normally always upgrade all packages to the latest (non-prerelease) version, this is currently the only exception on that rule.

Thank you!

**Comment by JimBobSquarePants** [comment]
We're currently targeting an RC1 release. You can see the remaining open issues here.

https://github.com/SixLabors/ImageSharp.Drawing/milestone/1

There's open PRs for a few of them but others still need work (though should be trivial enough) but any of them are up-for-grabs.
